zeppelin-server/pom.xml

```diff
@@ -37,7 +37,7 @@
     <!--library versions-->
     <commons.httpclient.version>4.3.6</commons.httpclient.version>
     <jersey.version>2.22.2</jersey.version>
-    <hadoop-common.version>2.6.0</hadoop-common.version>
+    <hadoop.version>2.7.3</hadoop.version>
```

Comment:
Renaming the key "hadoop-common.version" to "hadoop.version" to make it similar what is being used in zeppelin-zengine.